### DISCUSSION TITLE
Hide stashes when opening Neogit

### DISCUSSION THREAD

**Discussion by corpooo** [NeogitOrg/neogit#241] [Q&A]
I have a lot of stashes. And when I open Neogit see a list of them and it kinda gets confusing since I expect Neogit to be empty since I don't have any changes. Would it be possible to hide that section somehow?

**Comment by TimUntersberger** [comment]
@andreicorpo I opened an issue. This is currently not possible.

**Reply by corpooo** [reply]
Thanks! Really appreciate the fast response and the awesome work you've done so far with this plugin. 😄
